tests/run-make/no-input-file/rmake.rs

```diff
@@ -0,0 +1,9 @@
+extern crate run_make_support;
+
+use run_make_support::{diff, rustc};
+
+fn main() {
+    let output = rustc().arg("--print").arg("crate-name").run_fail();
```

Comment:
This should probably use

https://github.com/rust-lang/rust/blob/7f2fc33da6633f5a764ddc263c769b6b2873d167/src/tools/run-make-support/src/rustc.rs#L157-L167

to check that we don't ICE (whose error code is 101) but fail in a "warm" way (i.e. error code 1, error message "no input file given")